src/pykeen/nn/combinations.py

```diff
@@ -265,3 +339,9 @@ def forward(self, x: torch.FloatTensor, literal: torch.FloatTensor) -> torch.Flo
         z = self.gate_activation(self.gate_entity_layer(x) + self.gate_literal_layer(literal) + self.bias)
         h = self.linlayer_activation(self.combination_linear_layer(combination))
         return self.dropout(z * h + (1 - z) * x)
+
+
+combination_resolver: ClassResolver[NCombination] = ClassResolver.from_subclasses(
```

Comment:
these are both "combinations" but very different kinds of things. maybe we need to make more clever names

Reply:
like how about TensorCombination

Reply:
I think they are the same, just that the `NCombination` is a more general case with more than two representations.